tools/code_format/check_format.py

```diff
@@ -92,7 +92,8 @@
     "./source/server/admin/utils.cc", "./source/server/admin/stats_params.h",
     "./source/server/admin/stats_request.cc", "./source/server/admin/prometheus_stats.h",
     "./source/server/admin/prometheus_stats.cc", "./tools/clang_tools/api_booster/main.cc",
-    "./tools/clang_tools/api_booster/proto_cxx_utils.cc", "./source/common/version/version.cc")
+    "./tools/clang_tools/api_booster/proto_cxx_utils.cc", "./source/common/version/version.cc",
+    "./source/common/router/header_parser_utils.cc")
```

Comment:
this list just got moved to tools/code_format/config.yaml - you will need to adjust that one instead

Reply:
Thanks! Adjusted tools/code_format/config.yaml.